pkg/contracts/solana/gateway.go

```diff
@@ -54,3 +59,20 @@ func ParseGatewayIDAndPda(address string) (solana.PublicKey, solana.PublicKey, e
 
 	return gatewayID, pda, err
 }
+
+// ParseRentPayerPda parses the rent payer program derived address from the given string
+func ParseRentPayerPda(gatewayAddress string) (solana.PublicKey, error) {
```

Comment:
IMO this makes code more consistent

```suggestion
func RentPayerPDA(gateway solana.PublicKey) (solana.PublicKey, error) {
```